panopticum/management/commands/importer.py

```diff
@@ -0,0 +1,1085 @@
+                comp_ver_obj.qa_perf_tests_status = 'unknown'  # dont have info
+                comp_ver_obj.qa_perf_tests_notes = ''  # dont have info
+                # comp_version.qa_perf_tests_signoff = ''  # dont have info
+                comp_ver_obj.qa_longhaul_tests_status = 'unknown'  # dont have info
+                comp_ver_obj.qa_longhaul_tests_notes = ''  # dont have info
+                # comp_version.qa_longhaul_tests_signoff = ''  # dont have info
+                comp_ver_obj.qa_security_tests_status = 'unknown'  # dont have info
+                comp_ver_obj.qa_security_tests_notes = ''  # dont have info
+                # comp_version.qa_security_tests_signoff = ''  # dont have info
+                comp_ver_obj.qa_api_tests_status = 'unknown'  # dont have info
+                comp_ver_obj.qa_api_tests_notes = ''  # dont have info
+                # comp_version.qa_api_tests_signoff = ''  # dont have info
+                comp_ver_obj.qa_anonymisation_tests_status = 'unknown'  # dont have info
+                comp_ver_obj.qa_anonymisation_tests_notes = ''  # dont have info
+                # comp_version.qa_anonymisation_tests_signoff = ''  # dont have info
+                comp_ver_obj.qa_upgrade_tests_status = 'unknown'  # dont have info
+                comp_ver_obj.qa_upgrade_tests_notes = ''  # dont have info
+                # comp_version.qa_upgrade_tests_signoff = ''  # dont have info
+                # comp_version.meta_update_by = ''  # dont have info
+                # comp_version.meta_update_date = ''  # dont have info
+                comp_ver_obj.meta_deleted = False  # dont have info
+                # comp_ver_obj.meta_product_versions # Not filling this as we dont have information
+                # comp_ver_obj.meta_searchstr_locations # Not filling this as we dont have information
+
+                # Now we will fill details fetched from CSV
+                comp_ver_obj.component_id = comp_obj
+
+                maintainers = Utility.get_users_by_emailid(kwargs['owners'])
+                if maintainers is not None and len(maintainers) > 0:
+                    comp_ver_obj.owner_maintainer = maintainers[0]
+                    for maintainer in maintainers:
+                        comp_ver_obj.owner_expert.add(maintainer)
+                        comp_ver_obj.owner_escalation_list.add(maintainer)
+
+                product_managers = Utility.get_users_by_emailid(kwargs['product_managers'])
+                for pm in product_managers if product_managers is not None else []:
+                    comp_ver_obj.owner_product_manager.add(pm)
+
+                architects = Utility.get_users_by_emailid(kwargs['architects'])
+                for a in architects if architects is not None else []:
+                    comp_ver_obj.owner_architect.add(a)
+
+                # TODO: Check with AA how to get Program Manager list, escalation list.

```

Comment:
?

Reply:
Ignore this comment. The comment was added considering v1 csv format where we didn't have the information.

I will remove it in subsequent changes.